### DISCUSSION TITLE
private/how-to-use-fomepay

### DISCUSSION THREAD

**Discussion by giscus** [imarvinle/CSGuide#69] [Announcements]
# private/how-to-use-fomepay

Onlyfans支付、虚拟卡、onlyfans订阅、hongkongdoll订阅

https://csguide.cn/private/how-to-use-fomepay.html

**Comment by JasonGuo1** [comment]
fomepay已经跑路了，不能用了。

**Reply by imarvinle** [reply]
是的，fomepay 已经跑路了，大家请使用 wildcard，稳定安全：https://csguide.cn/private/chatgpt.html
